openupgrade_scripts/scripts/product/tests/data.py

```diff
@@ -0,0 +1,27 @@
+env = locals().get("env")
+# set specific pricelist
+pricelist_main = env["product.pricelist"].create(
+    {
+        "name": "Pricelist for main company",
+    }
```

Comment:
Removing trailing comma the auto-formatter saves you a lot of lines:

```suggestion
    {"name": "Pricelist for main company"}
```